src/pybind/mgr/cephadm/module.py

```diff
@@ -703,24 +704,23 @@ def _set_ssh_config(self, inbuf: Optional[str] = None) -> Tuple[int, str, str]:
         self._reconfig_ssh()
         return 0, "", ""
 
-    @orchestrator._cli_write_command(
-        prefix='cephadm clear-ssh-config',
-        desc='Clear the ssh_config file')
+    @orchestrator._cli_write_command('cephadm clear-ssh-config')
     def _clear_ssh_config(self) -> Tuple[int, str, str]:
         """
-        Clear the ssh_config file provided from stdin
+        Clear the ssh_config file
         """
+        # Clear the ssh_config file provided from stdin
         self.set_store("ssh_config", None)
         self.ssh_config_tmp = None
         self.log.info('Cleared ssh_config')
         self._reconfig_ssh()
         return 0, "", ""
 
-    @orchestrator._cli_read_command(
-        prefix='cephadm get-ssh-config',
-        desc='Returns the ssh config as used by cephadm'
-    )
+    @orchestrator._cli_read_command('cephadm get-ssh-config')
```

Comment:
we still have all API docs being broken on docs.ceph.com. I don't know if this change is going to break the automatically generated mon command API t(hat is broken right now anyway)

Reply:
@sebastian-philipp what do you mean by "all"? i think the only missing pieces of the API docs are 

- document generated using javadoc
- mon command doc generated using `gen_mon_command_api.py`. this part is missing from docs.ceph.com since its day 1. see https://github.com/ceph/ceph/pull/33886#issuecomment-613430619 .

tl;dr. yes. it's going to break the mon + mgr commands doc build.

long story: since `src/script/gen_static_command_descriptions.py` imports mgr modules and parses the `cmd` attribute of the elements in `COMMANDS`, and `cmd` is composed of `prefix` and `args` attributes of a command. the latter is rebuilt when a command is registered using the `CLICommand` decorator and its variants. so presumably, it won't be difficult to adapt this script to use `prefix` and `args` instead of `cmd`.

if you are fine with the direction of this change in general. i will do the adaption work. but i don't have plan to fix the broken mon command doc at this moment. i did some investigations though. a viable approach is to create a homebrew sphinx extension for generating the sphinx docs (not the rst files) from the code, like what autodoc does.

Reply:
for gen_mon_command_api, we could also simply commit the generated .rst file into the tree. We just need a make check test to verify the file is up to date. wdyt?

Reply:
#38807